installing/installing-preparing.adoc

```diff
@@ -39,7 +39,7 @@ endif::openshift-origin[]
 
 You can deploy an {product-title} 4 cluster to both on-premise hardware and to cloud hosting services, but all of the machines in a cluster must be in the same data center or cloud hosting service.
 
-If you want to use {product-title} but do not want to manage the cluster yourself, you have several managed service options. If you want a cluster that is fully managed by Red Hat, you can use link:https://www.openshift.com/products/dedicated/[OpenShift Dedicated] or link:https://www.openshift.com/products/online/[OpenShift Online]. You can also use OpenShift as a managed service on Azure, AWS, IBM Cloud VPC, or Google Cloud. For more information about managed services, see the link:https://www.openshift.com/products[OpenShift Products] page.
+If you want to use {product-title} but do not want to manage the cluster yourself, you have several managed service options. If you want a cluster that is fully managed by Red Hat, you can use link:https://www.openshift.com/products/dedicated/[OpenShift Dedicated] or link:https://www.openshift.com/products/online/[OpenShift Online]. You can also use OpenShift as a managed service on Azure, AWS, IBM Cloud VPC, or Google Cloud. For more information about managed services, see the link:https://www.openshift.com/products[OpenShift Products] page. {product-title} does not support cloud-based storage for bare metal installation.
```

Comment:
I'm not a native speaker and the new sentence does not make much sense to me. Why would I use cloud storage on a real on prem bare metal OCP installation? Can we somehow emphasize that this targets clusters that use a cloud VM as a (virtual) bare metal?

Reply:
@jsafrane Can you verify if the reworded statement sounds better?

Reply:
lgtm